esgf_utilities/esg_property_manager.py

```diff
@@ -5,7 +5,7 @@
 import logging
 import ConfigParser
 import yaml
-from configobj import ConfigObj
+from backports import configparser
```

Comment:
Did you confirm that the backported configparser maintains comments when writing to the properties file?  That was the main reason for using ConfigObj.  I do have an issue open for creating a guide for the options of the esgf.properties file but was using comments as an extra layer of instruction for users.

Reply:
It does not preserve them in the `/esg/config/esgf.properties` file. Do they need to be there?

Reply:
I was going to use the comments as a way to provide info on some of the more vague configurable options initially.  But I have this issue (#418) open that I plan to add to the documentation page.  That will supersede the comments in esgf.properties. So in short, your approach should be fine when I finish #418